packages/palette/src/elements/Button/Button.tsx

```diff
@@ -24,8 +24,6 @@ export interface ButtonProps
   loading?: boolean
   /** Uses inline style for button */
   inline?: boolean
-  /** This does nothing at all: TODO: Remove */
-  block?: boolean
```

Comment:
lol I kept setting this thinking it was going to do what it said it would (previously: make button full-width)